Fix color of settings icons

BlackyHawky opened the pull request:
Fixes #188 

| Before | After |
| :------: | :----: |
| <img width=200 src="https://github.com/Helium314/openboard/assets/139015663/fb528bb9-dc51-487a-bbc5-1c87604b8038"> | <img width=200 src="https://github.com/Helium314/openboard/assets/139015663/eb02b9d4-9a62-4e32-b70b-d7dfc1bc55c3"> |

_Tested on Huawei phone with Android 10 and Samsung tablet with Android 13_

Helium314 commented:
I adjusted the tint to be set when settings are loaded for old Android versions, so for the settings icons there is no need to care about the API < 21 warning any more.

Could you rename the `_white` files, as now _white_ is not correct any more?

BlackyHawky commented:
> I adjusted the tint to be set when settings are loaded for old Android versions, so for the settings icons there is no need to care about the API < 21 warning any more.
> 
> Could you rename the `_white` files, as now _white_ is not correct any more?

Thank you very much for working on this.
The files are now renamed.

Helium314 commented:
Thanks!
